tutorials/03_command_line.md

```diff
@@ -144,5 +142,5 @@ gz fuel edit -p -u https://fuel.gazebosim.org/1.0/OpenRobotics/worlds/Fortress%2
 Use the `-b` option to make a resource public. For example:
 
 ```
-gz fuel edit -b -u https://fuel.gazebosim.org/1.0/OpenRobotics/worlds/%20Demo --header 'Private-token: YOUR_TOKEN'
+gz fuel edit -b -u https://fuel.gazebosim.org/1.0/OpenRobotics/worlds/Fortress%20Demo --header 'Private-token: YOUR_TOKEN'
```

Comment:
```suggestion
gz fuel edit -b -u https://fuel.gazebosim.org/1.0/OpenRobotics/worlds/Garden%20Demo --header 'Private-token: YOUR_TOKEN'
```

Reply:
All the examples in this document use `Fortress Demo`. Any reason to change it to Garden since it's just showing how the `gz fuel` command works?